src/frontend/src/features/layouts/components/mailbox-settings/modal-mailbox-settings/_index.scss

```
@@ -0,0 +1,156 @@
+  // unfold chevron and a subtle hover fill, mirroring a macOS popup button.
+  &__dropdown-button {
+    border: none;
+    height: inherit;
+    justify-content: space-between;
+  }
+
+  &__dropdown-button__content {
+      display: flex;
+      align-items: center;
+      gap: var(--c--globals--spacings--s);
+      // Let the content shrink inside the (full-width) card so the text column
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)